include/deal.II/grid/grid_tools_cache.h

```diff
@@ -134,6 +157,14 @@ namespace GridTools
     const KDTree<spacedim> &get_vertex_kdtree() const;
 #endif
 
+#ifdef DEAL_II_WITH_MPI
+    /**
+     * Return a reference to the cached global_bounding_boxes object.
+     */
+    const std::vector< std::vector< BoundingBox<spacedim> > >
+    &get_global_bounding_boxes() const;
```

Comment:
Same here -- just make the function always available.

Reply:
Second, the doc string does not actually explain what the function does -- a user will not know what that variable is or represents. You should be describing what the function's return object *represents*.

I also think that the name of the function is not really descriptive. Does the function return each process's subdomain's bounding box?